web-registry/src/pages/EcocreditBatches/EcocreditBatches.tsx

```diff
@@ -0,0 +1,43 @@
+import { useEffect } from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
+import { Box } from '@mui/material';
+
+import Section from 'web-components/lib/components/section';
+import { Title } from 'web-components/lib/components/typography';
+
+import { CreditBatches } from 'components/organisms';
+import { usePaginatedBatches } from 'hooks/batches/usePaginatedBatches';
+
+export const EcocreditBatches = (): JSX.Element => {
+  const navigate = useNavigate();
+  const { page: routePage } = useParams();
+  const { batchesWithSupply, setPaginationParams, paginationParams } =
+    usePaginatedBatches();
+  const { page } = paginationParams;
+
+  useEffect(() => {
```

Comment:
I'm not sure I understand the advantages of tying a table page to a route. Can you explain?

Reply:
Sure, the main advantages of having the table page tied to the route are:
- to link directly to a specific page batch (facilitate saving and sharing of URLs)
- allow SEO bots to crawl the batches